enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -639,7 +639,7 @@ func NewContainer(ctx context.Context, env environment.Env, task *repb.Execution
 				log.CtxInfof(ctx, "Failed to get VM snapshot for keyset %s: %s", snaploader.KeysetDebugString(ctx, c.env, c.SnapshotKeySet(), c.supportsRemoteSnapshots), err)
 			} else {
 				label = metrics.HitStatusLabel
-				log.CtxInfof(ctx, "Found snapshot for key %s", snaploader.KeyDebugString(ctx, c.env, snap.GetKey(), c.supportsRemoteSnapshots))
+				log.CtxInfof(ctx, "Found snapshot with ID %q for key %s", snap.GetVMMetadata().GetSnapshotId(), snaploader.KeyDebugString(ctx, c.env, snap.GetKey(), c.supportsRemoteSnapshots))
```

Comment:
Could you add this to snaploader.KeyDebugString instead so it applies everywhere?

Reply:
Done and renamed to `SnapshotDebugString` since the ID isn't part of the key

Reply:
It is a part of the key? Not sure why the field isn't ordered so maybe you missed it? 

https://github.com/buildbuddy-io/buildbuddy/blob/e8f1218846a4e093080c36aa1cbb8acbe58699ce/proto/firecracker.proto#L56

Reply:
It's only part of the key when calling `enterprise/tools/vmstart` - we usually leave that field blank and match based on branch info + platform etc.